src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextFieldPredefinedListEditorSettings.Edit.cshtml

```diff
@@ -77,25 +84,35 @@
             </div>
         </div>
     </div>
-</script>
+</script> *@
 
 <div id="@Html.IdFor(m => m)" class="field-editor field-editor-predefinedlist">
-    <div class="row">
-        <div class="mb-3 col-sm-6">
-            <label asp-for="Editor" class="form-label">@T["What type of list editor should be used?"]</label>
-            <select asp-for="Editor" class="form-select">
-                <option value="@nameof(EditorOption.Dropdown)">@T["Drop down list"]</option>
-                <option value="@nameof(EditorOption.Radio)">@T["Radio buttons"]</option>
-            </select>
-        </div>
-    </div>
+	<div class="row">
+		<div class="mb-3 col-sm-6">
+			<label asp-for="Editor" class="form-label">@T["What type of list editor should be used?"]</label>
+			<select asp-for="Editor" class="form-select">
+				<option value="@nameof(EditorOption.Dropdown)">@T["Drop down list"]</option>
+				<option value="@nameof(EditorOption.Radio)">@T["Radio buttons"]</option>
+			</select>
+		</div>
+	</div>
 
-    <div class="row">
-        <div class="col-sm-6">
-            <label asp-for="Options" class="form-label">@T["Options"]</label>
-            <a href="javascript:void(0)" v-on:click="showModal" class="float-end" title="@T["Edit Data"]"><i class="fa-solid fa-edit" aria-hidden="true"></i></a>
-            <options-table :data="sharedState"></options-table>
-            <options-modal :data="sharedState" v-on:reload-data="reloadData()"></options-modal>
-        </div>
-    </div>
+	<div class="row">
+		<div class="col-sm-6" >
+			<label asp-for="Options" class="form-label">@T["Options"]</label>
+			@* <options-table :data="sharedState"></options-table>
+			<options-modal :data="sharedState" v-on:reload-data="reloadData()"></options-modal> *@
+			<Component type="typeof(OptionEditor)" render-mode="WebAssemblyPrerendered" param-optionsjson="Model.Options" />
+			<input class="form-control" id="@Html.IdFor(m => m.Options)" name="@Html.NameFor(m => m.Options)" type="hidden" value="@Model.Options" />
+		</div>
+	</div>
 </div>
+<script at="Foot">
```

Comment:
We really need to have a bundler to start having script type="module" and get rid of these template scripts.